### PR TITLE
ci: Remove some unused packages

### PR DESCRIPTION
**Pull request by novakne**
Add quiet flag to wget

### CODE REVIEW

File: .builds/alpine.yml [hunk 1]
```diff
@@ -16,7 +16,6 @@ packages:
   - xwayland
   - meson
   - scdoc
-  - tar
```
Comment (ifreund) [inline .builds/alpine.yml:19]: it seems strange to remove this when we use tar below to extract the zig tarball...

Comment (novakne) [reply .builds/alpine.yml:19]: I imagine it is already a deps of something, I saw that is required by abuild, the program to build Alpine package.
I've been using it like so on my project and also for nfm without any problem, and as you can see the ci for this PR is succesfull, but if you prefer I can add it back.

Comment (tiosgz) [reply .builds/alpine.yml:19]: I believe that you should explicitly install/require everything that you explicitly use. (And leave implicit everything that you don’t use explicitly.)

Comment (ifreund) [reply .builds/alpine.yml:19]: Yeah, I'd prefer to keep everything we directly use explicitly listed. That way our CI doesn't break if the transitive dependencies brought in change.

**Pull request closed without merging**